renovate-config.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "description": "Default configuration for repositories in the Laminas organisation",
+    "extends": [
+        ":enableRenovate",
+        ":ignoreModulesAndTests",
+        ":automergeDisabled",
+        ":automergeBranch",
+        ":prNotPending",
+        ":rebaseStalePrs",
+        ":separateMajorReleases",
+        ":combinePatchMinorReleases",
+        ":enableVulnerabilityAlerts",
+        ":timezone(UTC)",
+        ":label(renovate)",
+        "group:allNonMajor",
+        "schedule:daily"
+    ],
+    "enabledManagers": ["composer"],
+    "lockFileMaintenance": {
```

Comment:
lockfile maintenance needs to be set on a timed schedule, i think. Otherwise it would be noisy

Reply:
Yep. The whole of renovate is set to run daily using the `"schedule:daily"` preset.

Reply:
There would be no PR for lockfile maintenance, these would be automatically merged without a PR being generated, should all checks pass. A PR will only be raised for updating a version constraint outside of its allowed range, or if any of the checks fail.

Reply:
I'm not the one getting these notifications though, so I'll make sure weekly vs daily is part of what needs to be decided.

Reply:
sorry, it is literally in PR description.

Reply:
I've added a list of outstanding questions that need to be decided on my maintainers, I've added daily vs weekly to this.